bin/cheribsdtest/cheribsdtest_fault.c

```diff
@@ -220,6 +220,14 @@ test_nofault_cfromptr(const struct cheri_test *ctp __unused)
 	char * __capability cd; /* stored into here */
 
 	cb = cheri_ptr(buf, 256);
+#if defined(__aarch64__)
+	/*
+	 * morello-llvm emits cvtz for this intrinsic, which has an
+	 * address interpretation by default (unlike CFromPtr, which
+	 * has an offset interpretation).
+	 */
+	cd = __builtin_cheri_cap_from_pointer(cb, (vaddr_t)buf + 10);
```

Comment:
We should probably open an issue. Intrinsics should have the same behaviour across architectures.

Reply:
https://git.morello-project.org/morello/llvm-project/-/issues/16